tests/spec/float_series_ops/test_arithmetic_ops.py

```diff
@@ -58,3 +58,35 @@ def test_multiply_with_constant(spec_test):
         10.0 * p.f2,
         {1: 10.0 * 111.5, 2: None},
     )
+
+
+def test_truedivide(spec_test):
+    spec_test(
+        table_data,
+        p.f1 / p.f2,
+        {1: 101.3 / 111.5, 2: None},
+    )
```

Comment:
A further thought on testing here. Given that division is quite a lot more complex than any of the other operations and there are many more edges cases to cover I wonder if it's worth breaking out a separate `test_division_ops.py` file with its own `table_data`.

Reply:
that's probably a good idea